pagination.md

```diff
@@ -0,0 +1,178 @@
+# Pagination
+
+This document describes a mechanism by which a server can return a set of
+records to a client in an incremental fashion. Often this will be used when
+a client is doing a query for a set of records and the result set is too large
+to return in one response.
+
+## Notations and Terminology
+
+### Notational Conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+## Client Request
+
+When a client sends a request to a server which is meant to result in a
+set of records being returned, the client MAY include extra attributes
+to control how those records are to be returned if the results can not
+fit within one response message.
+
+### Client Attributes
+
+The follow list of attributes MAY be included in a client's request for
+a set of records from a server. The client MUST only specify these on
+the initial request for set. If the results can not fit within one
+message then these attributes MUST NOT be added to subsequent requests
+by the client. Note, the server MAY include these attributes within
+the URI-reference returned in a response message, but the client MUST NOT
+modify those values.
+
+#### size
```

Comment:
suggestion: limit (size) and offset (page) are more commonly used in pagination queries

Reply:
ok done